traffic_portal/app/src/common/api/ServiceCategoryService.js

```diff
@@ -20,7 +20,7 @@
 var ServiceCategoryService = function($http, ENV, locationUtils, messageModel) {
 
     this.getServiceCategories = function(queryParams) {
-        return $http.get(ENV.api['root'] + 'servicecategories', {params: queryParams}).then(
+        return $http.get(ENV.api['latest'] + 'servicecategories', {params: queryParams}).then(
```

Comment:
https://github.com/apache/trafficcontrol/blob/master/traffic_portal/app/src/scripts/config.js

latest is now gone. you'll want to set these back to root.